.github/workflows/ci-gh1.yml

```diff
@@ -0,0 +1,63 @@
+name: Build legate.core on GH
+
+concurrency:
+  group: ci-gpu-on-${{ github.event_name }}-from-${{ github.ref_name }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches-ignore:
+      - gh-pages  # deployment target branch (this workflow should not exist on that branch anyway)
+  # NOTE: this workflow will not work for pull_requests since GH pull requests do not have access to secrets.
+
+jobs:
+  build:
+    permissions: read-all
+    # Ref: https://docs.rapids.ai/resources/github-actions/#cpu-labels for `linux-amd64-cpu4`
+    runs-on: ${{ github.repository == 'nv-legate/legate.core' && 'linux-amd64-cpu4' || 'ubuntu-latest' }}
+    container:
+      options: -u root
+      image: pauletaylor/devcontainers:23.06-cpp-cuda11.8-mambaforge-ubuntu22.04
+      env:
+        DEFAULT_CONDA_ENV: legate
+        PYTHONDONTWRITEBYTECODE: 1
+        SCCACHE_REGION: us-east-2
+        SCCACHE_BUCKET: rapids-sccache-devs
+        SCCACHE_S3_KEY_PREFIX: legate-cunumeric-dev
+        VAULT_HOST: ${{ secrets.PERSONAL_ACCESS_TOKEN && 'https://vault.ops.k8s.rapids.ai' || '' }}
+        GH_TOKEN: "${{ secrets.PERSONAL_ACCESS_TOKEN || secrets.GITHUB_TOKEN }}"
+        GITHUB_TOKEN: "${{ secrets.PERSONAL_ACCESS_TOKEN || secrets.GITHUB_TOKEN}}"
+      volumes:
+        - ${{ github.workspace }}/out:/tmp/out
+    steps:
+      - name: Checkout legate.core (= this repo)
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          path: legate
+
+      - name: Setup
+        shell: bash -eo pipefail {0}
+        run: |
+          apt update;
+          DEBIAN_FRONTEND=noninteractive \
+          apt install -y --no-install-recommends \
+              numactl rlwrap;
+
+          mv legate/continuous_integration/home/coder/.gitconfig /home/coder/.gitconfig;
+          mv legate/continuous_integration/home/coder/.local /home/coder/.local;
```

Comment:
Is this file missing in the pull request? Or maybe I am just looking in the wrong place.